packages/ember-data/lib/adapters/rest-adapter.js

```diff
@@ -391,7 +391,7 @@ export default Adapter.extend(BuildURLMixin, {
     @return {Promise} promise
   */
   findQuery: function(store, type, query) {
-    var url = this.buildURL(type.typeKey, null, null, 'findQuery');
+    var url = this.buildURL(type.typeKey, query, null, 'findQuery');
```

Comment:
That would be a breaking change for anyone currently overriding `buildURL` that is expecting it to be an actual snapshot. And I would say that the breaking change should be adding a formal `query` argument so that the signature would be:

```js
buildURL: function(type, id, query, snapshot, requestType) {
```